src/types/common.ts

```diff
@@ -58,20 +64,61 @@ export type Position = {
 };
 
 export type StreamingFeeInfo = {
+  /**
+   * Address to accrue fees to
+   */
   feeRecipient: string;
+  /**
+   * Percent of Set accruing to manager annually (1% = 1e16, 100% = 1e18)
+   */
   streamingFeePercentage: BigNumber;
+  /**
+   * The amount of streaming fees that haven't been actualized to the fee receipient yet
+   */
   unaccruedFees: BigNumber;
 };
 
 export type SetDetailsWithStreamingInfo = {
+  /**
+   * Set name (e.g. "DeFi Pulse Index")
+   */
   name: string;
+  /**
+   * Set symbol (e.g. "DPI")
+   */
   symbol: string;
+  /**
+   * The address of the set manager
+   */
   manager: string;
+  /**
+   * List of module addresses that have been abled on this set
+   */
   modules: Address[];
+  /**
+   * Status of each module represented as 0, 1, or 2. Every element in this list
+   * corresponds to an element of the same index in modules.
+   *
+   * 0 => NONE
+   * 1 => PENDING
+   * 2 => INITIALIZED
```

Comment:
This could use some fact checking. I based this off 
1. https://github.com/SetProtocol/set-protocol-v2/blob/24acd74aff8c4fbce39f6253035caa237ef41436/contracts/protocol-viewers/SetTokenViewer.sol#L100
1. https://github.com/SetProtocol/index-coop-contracts-legacy/blob/c4bccb2f476e86b74d6c0bfbcaabec0cf8297265/external/contracts/set/SetToken.sol#L119-L121

but don't have first-hand experience w/ this field yet

Reply:
This looks correct. Thanks for this @rootulp !